src/doc/trpl/the-stack-and-the-heap.md

```diff
@@ -525,6 +525,13 @@ that don’t make it true 100% of the time, too. Rather than relying on the stac
 and `Drop` to clean up memory, the garbage collector deals with the heap
 instead.
 
+C, on the other hand, does nothing at all. The programmer must explicitly
+deallocate all heap-allocated memory with the `free()` function. While this
+grants him a lot of liberty as to when he allocates and deallocates data, it
```

Comment:
“programmers” on the first line and “them” here, making this sentence gender-agnostic..